pkg/apis/flow/v1alpha1/flow.proto

```diff
@@ -141,6 +141,8 @@ message Kubernetes {
   string egress_node_name = 32;
   string egress_node_uid = 33;
   string egress_uid = 34;
+
+  bytes destination_service_ip = 35;
```

Comment:
can you look at the possibility of marking `destination_cluster_ip` as `deprecated`?
note that we don't want to remove the field for a couple of versions as we have to support version skew between FE and FA